packages/p2p-v2/src/pages/advertiser/screens/Advertiser/Advertiser.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+import { PageReturn, ProfileContent } from '@/components';
+import { LabelPairedEllipsisVerticalLgRegularIcon } from '@deriv/quill-icons';
+import { useDevice } from '@deriv-com/ui';
+import { AdvertiserAdvertsTable } from '../AdvertiserAdvertsTable';
+import './Advertiser.scss';
+
+const Advertiser = () => {
+    const { isMobile } = useDevice();
+    const history = useHistory();
+    const location = useLocation();
+    const urlParams = new URLSearchParams(location.search);
+    const advertiserId = urlParams.get('id') ?? undefined;
```

Comment:
it gives me null, and I need to have it as undefined, since in the ProfileContent and AdvertiserAdvertsTable components, im passing it down into the payload for the API calls, and need it to be undefined, else it will be stuck if i leave it as null